raster/src/main/scala/geotrellis/raster/io/geotiff/StorageMethod.scala

```diff
@@ -51,3 +53,47 @@ object Striped extends StripedStorageMethod {
 
   implicit def objectToStorageMethod(s: StripedStorageMethod): Striped = Striped()
 }
+
+object StorageMethod {
+  implicit val storageMethodDecoder: Decoder[StorageMethod] =
+    new Decoder[StorageMethod] {
+      final def apply(c: HCursor): Decoder.Result[StorageMethod] = {
+        for {
+          storageType <- c.downField("storageType").as[String]
+        } yield {
+          storageType match {
+            case "striped" => {
+              val rowsPerStrip =
+                c.downField("rowsPerStrip").as[Option[Int]] match {
+                  case Left(_)  => None
+                  case Right(v) => v
+                }
+              new Striped(rowsPerStrip)
+            }
+            case _ => {
+              val cols = c.downField("cols").as[Int] match {
+                case Left(_)  => 256
```

Comment:
It is not true, such logic is fine for the application, but not for the library. A GDAL fallback behavior is the following: It is not tiled => it is striped.